runtime/src/lib.rs

```diff
@@ -560,14 +559,25 @@ impl cumulus_pallet_aura_ext::Config for Runtime {}
 
 parameter_types! {
     pub const MaxInboundSuspended: u32 = 1000;
+	/// The asset ID for the asset that we use to pay for message delivery fees.
+	pub FeeAssetId: AssetId = AssetId(RelayLocation::get());
+	/// The base fee for the message delivery fees.
+	pub const ToSiblingBaseDeliveryFee: u128 = CENTS.saturating_mul(3);
```

Comment:
is there any specific reason why we chose `3`? Maybe we should comment on that as inline docs?

Reply:
It was taken from Kusama config as a reference. I can mention Kusama in the comment.